templates/cohorts/new_cohort.html

```diff
@@ -195,6 +195,24 @@ <h4 class="panel-title pull-left">Selected Filters</h4>
             </div>
         </div>
 
+        <div class="cohort-info panel panel-default">
+            <div class="panel-heading">
+                <h4 class="panel-title">Details</h4>
+            </div>
+            <div class="panel-body">
```

Comment:
Just the [bootstrap](http://getbootstrap.com/css/#grid-example-basic) documentation. You may need to add a .row class to the .panel div, but check out the documentation if you're not familiar with the grid system. Or just try adding .col-6-md to each inline-detail instead. 

I'd just prefer to use the grids when possible.